cloudant-sync-datastore-core/src/main/java/com/cloudant/sync/util/JSONUtils.java

```diff
@@ -34,7 +34,9 @@
 
     private static final String LOG_TAG = "JSONUtils";
 
-    public static final byte[] EMPTY_JSON = "{}".getBytes(Charset.forName("UTF-8"));
+    public static byte[] emptyJSONAsBytes(){
+        return "{}".getBytes(Charset.forName("UTF-8"));
+    }
```

Comment:
Done in dbec83e however I opted to use `Arrays.copyOf` because I like using less lines :)